### DISCUSSION TITLE
HTML5 <dialog> Element

### DISCUSSION THREAD

**Discussion by techboyg5** [twbs/bootstrap#31656] [General]
Any way that this element could be used for Bootstrap modals in the future?

**Comment by mdo** [comment]
Please use discussions for open ended questions.

**Comment by patrickhlauke** [comment]
also, as support is still not all that pervasive...i'd say it's best to hold off https://caniuse.com/?search=dialog

**Reply by patrickhlauke** [reply]
irrelevant if we then still need to polyfill it for browsers that don't do it natively yet

**Comment by mkurz** [comment]
With the recent release of Safari 15.4 all major browser now support the <dialog> element natively:
* https://caniuse.com/dialog
* https://developer.apple.com/documentation/safari-release-notes/safari-15_4-release-notes
* https://webkit.org/blog/12445/new-webkit-features-in-safari-15-4/

**Comment by WinterSilence** [comment]
@mkurz @techboyg5 https://developer.mozilla.org/en-US/docs/Web/API/HTMLDialogElement#browser_compatibility

**Comment by mdo** [comment]
With the improved browser support, we may be able to implement it in a v6 when we we get around to that. Not 100% sure just yet. All depends on how many browser versions we drop between now and then.

**Reply by WinterSilence** [reply]
@mdo top5 Android browsers
